src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -290,12 +299,16 @@ object MavenCoordinate {
 
   def parse(s: String): ValidatedNel[String, MavenCoordinate] =
     s.split(":") match {
+      case Array(g, a, _, c, v) => Validated.valid(MavenCoordinate(MavenGroup(g), MavenArtifactId(a), Classifier(c), Version(v)))
```

Comment:
I think you want that to be `Array(g, a, "jar", c, v)` to match the above, no? Can you comment why it is safe to ignore it?

Reply:
Yeah, that's fair. I think my thinking at the time was "I don't care what it is, because we're ignoring it everywhere." I'll change it to be more explicit (which will make it fail more nicely at the end of this `match`).